lib/hathitrust/subjects.rb

```diff
@@ -0,0 +1,61 @@
+## frozen_string_literal: true
+
+module Traject::Macros::HathiTrust
+  module Subject
+
+    LOWER_LETTER_PAT = /\p{Lower}/
+    STRIP_PUNCT_PAT = /\A\p{Punct}*(.*?)\p{Punct}*\Z/
+
+    # Strip leading/trailing punctuation
+    # @param [String] string of a subject
+    # @return [String] The normalized subject string
+    def self.normalize_subject(str)
+      STRIP_PUNCT_PAT.match(str)[1].gsub("/\s+/", ' ')
+    end
+
+    # Determin the 880 (linking fields) for the given field. Should probably be pulled
+    # out into a more generically-available macro
+    # @param [MARC::Record] r The record
+    # @param [MARC::DataField] f The field you want to try to match
+    # @param [Array<MARC::DataField>] A (possibly empty) array of linked fields
+    def self.eight_eighties_for(r, f)
```

Comment:
Could we call this "alternate graphic representations" or something else that reveals the purpose of the 880?

Might be helpful to provide some examples in the comments.